incubator/distribution/templates/distribution-deployment.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "distribution.fullname" . }}
+  labels:
+    app: {{ template "distribution.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

Comment:
The latest `_helpers.tpl` you get with `helm create` has a `chart` partial. Please update accordingly.